### PR TITLE
Added .DS_Store files to Git Ignore

### PR DESCRIPTION
**Pull request by nsuchy**
This pull request adds .DS_Store files to Git Ignore, developers on macOS may accidentally commit these local files to the Git repo - it would make sense to add it to Git Ignore to accommodate for those developers. If the maintainers agree with this change, it should be merged.

### CODE REVIEW

**Comment by jimtryon** [comment]
I agree, this would be useful for those who develop in Mac environments.

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,7 @@ node_modules
 .sass-cache/
 public/
 .cache/
+
+# .DS_Store Files
```
Comment (Bouncey) [inline .gitignore]: Ok, so this is an error I made. Yes I agree with addin `.DS_Store` to the `.gitignore` file, but I had an area set aside for custom ignores, I just didn't mark it out correctly.

Lines 1-4 are our custom ignores for this repo, the rest were added by `Gatsby`

Can you add a comment at the top of the file donating custom ignores and move `.DS_Store` there please

Many thanks

**Comment by nsuchy** [comment]
@Bouncey The commit has been ammended - please review and tell me if further changes are needed.

**Comment by Bouncey** [comment]
## LGTM :+1: 

Thanks for contributing :sparkles:

**Pull request merged**